Pipeline/CreateUCDComponentVersion/dbb-ucd-packaging.groovy

```diff
@@ -495,20 +626,38 @@ if (!properties.preview.toBoolean()){
 }
 
 /**
- * parse data set name and member name
- * @param fullname e.g. BLD.LOAD(PGM1)
- * @return e.g. (BLD.LOAD, PGM1)
+ * return true if the file is a file stored on zFS, false otherwise
  */
-def getDatasetName(String fullname){
-	def ds,member;
-	def elements =  fullname.split("[\\(\\)]");
-	ds = elements[0];
-	member = elements.size()>1? elements[1] : "";
-	return [ds, member];
+def isZFSFile(String name) {
+    Path path = Paths.get(name);
+    return Files.exists(path);
 }
 
 /**
+ * Extract the directory and the file 
+ * from the fullname of a zFS file
+ * For instance: /var/test/file.txt  --> [/var/test, file.txt]
  */
+def extractDirectoryAndFile(String fullname) {
```

Comment:
I think this should rather be a `get` Method instead of `extract`.

Reply:
Done